### DISCUSSION TITLE
Do we have Atlassian Confluence support on Dispatch ?

### DISCUSSION THREAD

**Discussion by sasikanthpamuru** [Netflix/dispatch#1650] [Ideas]
Today we have support to Google Docs for creating documents. We started using dispatch and since we are tightly integrated with Atlassian Suite , we are planning to work on confluence plugin. anyone had spent time on this work already ? Any thoughts in general ?

**Comment by vikassinha2019** [comment]
Considering Confluence being used widely in different organisation for documentation purposes, it will be an important and useful feature for many.

**Comment by Cinojose** [comment]
I've created a pr on that.
https://github.com/Netflix/dispatch/pull/2940
